### PR TITLE
[P4-2108] Amend Prison labelling

### PR DESCRIPTION
**Pull request by merlinc**
## Proposed changes
Change the labelling from Prison to Prison or Youth Offender Institution

### Issue tracking

- [P4-2108](https://dsdmoj.atlassian.net/browse/P4-2108)

## Screenshots
Screenshot from 2020-09-04 16-35-20

#### Automated testing

- [ ] Added unit tests to cover changes
- [ ] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [ ] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

### Other considerations

Commit messages with a `fix` or `feat` type are automatically used to generate the [changelog](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/CHANGELOG.md) and
[GitHub release notes](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/releases) during the release task. Please make sure they will read well on their own in a
summary of changes and that the commit body gives a more detailed description of those changes if necessary.

- [x] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics
- [ ] Update [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/README.md) with any new instructions or tasks

### CODE REVIEW

File: locales/en/fields.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "short_label": "To",
     "items": {
       "prison_transfer": {
-        "label": "Prison"
+        "label": "Prison or Young Offender Institution"
```
Comment (teneightfive) [inline locales/en/fields.json:68]: I wonder if we should we be consistent with casing as below we write all in lowercase, but here we use capitals to suggest an acronym. Looking on GOV.UK, the [description for Feltham](https://www.gov.uk/guidance/feltham-yoi) uses all lowercase and includes the acronym:

> Feltham is a young offender institution (YOI) in London for boys and young men aged 15 to 21.

I wonder if we can use this pattern for the radio label and then use the acronym on the text label as the acronym has now been defined on the page:

```
Prison or young offender institution (YOI)
    Name of prison or YOI
```

Comment (solidgoldpig) [reply locales/en/fields.json:68]: The caps in the radio label vs the lowercase in the conditional are definitely a bit weirdy-beardy

Comment (merlinc) [reply locales/en/fields.json:68]: @teneightfive That looks like a good suggestion - GOV.UK is prior art, and those changes deal with the "weirdy beardy" case issues.

**Review by teneightfive — APPROVED** [review]

**Comment by jason-sims** [comment]
👍 LGTM. As discussed tweaking the content to align with GDS.

**Pull request merged**